### DISCUSSION TITLE
Last Name Auto Complete

### DISCUSSION THREAD

**Discussion by DavidSimonRothschild** [community/community#121072] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I noticed that github copilot automatically adds random surnames when you enter your first name in a script head.

I see this as a violation of privacy.

**Comment by davecheney** [comment, marked as answer]
@DavidSimonRothschild thank you for reporting this. Please contact support, https://support.github.com/contact who can escalate it to the appropriate development team.
